crowdin_api/api_resources/bundles/resource.py

```diff
@@ -20,15 +20,16 @@ def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None):
 
         return f"projects/{projectId}/bundles"
 
-    def get_bundles_exports_path(self, projectId: int, bundleId: int, exportId: Optional[str] = None):
+    def get_bundles_exports_path(
+        self, projectId: int, bundleId: int, exportId: Optional[str] = None
+    ):
         bundles_path = self.get_bundles_path(projectId, bundleId)
         if exportId:
             return f"{bundles_path}/exports/{exportId}"
         return f"{bundles_path}/exports"
 
     def list_bundles(
         self,
-        projectId: int,
```

Comment:
We can't change the interface of the current method that currently accepts the project ID because it'll be backward incompatible. Let's try making the `projectId` optional and populate it with the value passed to the `PROJECT_ID` option if present